test/e2e/bots_test.go

```diff
@@ -173,6 +173,7 @@ func runBotTest(t *testing.T,
 	err = waitForDeploymentReady(deployNsCli, appCfg.Deployment.Name, appCfg.Deployment.WaitTimeout)
 	require.NoError(t, err)
 
+	time.Sleep(appCfg.Slack.MessageWaitTimeout)
```

Comment:
Could you please describe why the sleep is necessary? I didn't see any issues with Slack tests recently 🤔

Reply:
Discord tests were failing haphazardly, this helped a little bit.

Reply:
> (BTW do we need separate MessageWaitTimeout for both platforms? If the test scenario is unified, maybe it would make sense to have just appCfg.MessageWaitTimeout 🤔 )

We do, I'll update with a follow up tiny PR after this is merged.